src/databases/Silo/avtSiloFileFormat.C

```diff
@@ -16635,12 +16635,12 @@ AddAle3drlxstatEnumerationInfo(avtScalarMetaData *smd)
     smd->AddEnumNameValue("RLX_SALE_25",25);
     smd->AddEnumNameValue("RLX_AdvTime_26",26);
     smd->AddEnumNameValue("RLX_HoldUntilActive_27",27);
-    smd->AddEnumNameValue("RLX_SlaveExtension_29",29);
+    smd->AddEnumNameValue("RLX_WorkerExtension_29",29);
```

Comment:
These enumeration values were taken from an old version of ALE3D. Its likely these as well as several others no longer make any sense.

Reply:
I believe this change is as *wrong* as the previous code was likely to be so am fine with leaving it.